slider/library.py

```diff
@@ -279,6 +279,29 @@ def lookup_by_md5(self, beatmap_md5):
         """
         return self._read_beatmap(self, beatmap_md5=beatmap_md5)
 
+    def beatmap_from_file(self, path, copy=False):
```

Comment:
Can we change this to `beatmap_from_path`? The idea is the distinguish between a function that takes a file path and one that takes an open file object.